tracer/src/Datadog.Trace/AppSec/SecuritySettings.cs

```diff
@@ -78,8 +78,13 @@ public static SecuritySettings FromDefaultSources()
             return new SecuritySettings(source);
         }
 
-        private int ParseWafTimeout(string wafTimeoutString)
+        private static int ParseWafTimeout(string wafTimeoutString)
         {
+            if (wafTimeoutString == null)
```

Comment:
nit: should we make this string.IsNullOrEmpty/string.IsNullOrWhiteSpace, seeing as the end result is the same AFAICT?

Reply:
If you like, I guess returning earlier is better 🤷 Fixed in d2d3455c1.